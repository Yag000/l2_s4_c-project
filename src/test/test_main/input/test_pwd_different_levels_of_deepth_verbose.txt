mkdir /t
mkdir /t/tt
mkdir /t/tt/ttt
mkdir /t/tt/ttt/tttt
mkdir /t/tt/ttt/tttt/ttttt
mkdir /t/tt/ttt/tttt/ttttt/tttttt
mkdir /t/tt/ttt/tttt/ttttt/tttttt/ttttttt
mkdir /t/tt/ttt/tttt/ttttt/tttttt/ttttttt/tttttttt
cd /t/tt/ttt/tttt/ttttt/tttttt/ttttttt/tttttttt
pwd
cd ..
pwd
cd ..
pwd
cd ..
pwd
cd ..
pwd
cd ..
pwd
cd ..
pwd
cd ..
pwd
cd ..
pwd
cd ..
pwd
mv /t /tests
cd /tests
pwd
