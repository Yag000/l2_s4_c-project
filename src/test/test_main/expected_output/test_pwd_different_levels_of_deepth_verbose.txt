Parsing file src/test/test_main/input/test_pwd_different_levels_of_deepth_verbose.txt ...
/$ mkdir /t
/$ mkdir /t/tt
/$ mkdir /t/tt/ttt
/$ mkdir /t/tt/ttt/tttt
/$ mkdir /t/tt/ttt/tttt/ttttt
/$ mkdir /t/tt/ttt/tttt/ttttt/tttttt
/$ mkdir /t/tt/ttt/tttt/ttttt/tttttt/ttttttt
/$ mkdir /t/tt/ttt/tttt/ttttt/tttttt/ttttttt/tttttttt
/$ cd /t/tt/ttt/tttt/ttttt/tttttt/ttttttt/tttttttt
/t/tt/ttt/tttt/ttttt/tttttt/ttttttt/tttttttt$ pwd
/t/tt/ttt/tttt/ttttt/tttttt/ttttttt/tttttttt
/t/tt/ttt/tttt/ttttt/tttttt/ttttttt/tttttttt$ cd ..
/t/tt/ttt/tttt/ttttt/tttttt/ttttttt$ pwd
/t/tt/ttt/tttt/ttttt/tttttt/ttttttt
/t/tt/ttt/tttt/ttttt/tttttt/ttttttt$ cd ..
/t/tt/ttt/tttt/ttttt/tttttt$ pwd
/t/tt/ttt/tttt/ttttt/tttttt
/t/tt/ttt/tttt/ttttt/tttttt$ cd ..
/t/tt/ttt/tttt/ttttt$ pwd
/t/tt/ttt/tttt/ttttt
/t/tt/ttt/tttt/ttttt$ cd ..
/t/tt/ttt/tttt$ pwd
/t/tt/ttt/tttt
/t/tt/ttt/tttt$ cd ..
/t/tt/ttt$ pwd
/t/tt/ttt
/t/tt/ttt$ cd ..
/t/tt$ pwd
/t/tt
/t/tt$ cd ..
/t$ pwd
/t
/t$ cd ..
/$ pwd
/
/$ cd ..
/$ pwd
/
/$ mv /t /tests
/$ cd /tests
/tests$ pwd
/tests
